src/components/HabitForm/HabitForm.js

```diff
@@ -0,0 +1,61 @@
+import "./HabitForm.css"
+import { AppContext } from "../../utils/context.js"
+import { QUERY_HABITS } from "../../utils/graph_queries"
+import { useQuery } from "@apollo/client"
+import { useEffect, useContext } from "react"
+import HabitCard from "../HabitCard/HabitCard"
+import { useMutation } from "@apollo/client"
+import { SUBMIT_HABIT } from "../../utils/graph_mutations"
+import "../../assets/icons/habit7-uncheck.png"
+
+const HabitForm = () => {
+  const { loading, error, data } = useQuery(QUERY_HABITS)
+  const { userHabits, setUserHabits, checkedHabitId } = useContext(AppContext)
+  const [createHabitEntry] = useMutation(SUBMIT_HABIT)
+
+  // const [checkedCards, setCheckedCards] = useState([])
+
+  useEffect(() => {
+    if (!loading && data) {
+      setUserHabits(data.fetchHabit)
```

Comment:
I don't know if the query for the habits is not working or the habits are not setting into state, but I am not seeing the habit buttons and the userHabits array is empty.

Reply:
Resovled